.github/workflows/test.yaml

```
@@ -28,6 +28,38 @@ jobs:
       - id: start-time-output
         run: echo "::set-output name=start-time::$(date)"
 
+  # # From https://github.com/marketplace/actions/skip-duplicate-actions
+  # # This action cleans up previously running instances of a workflow on the same branch.
+  # # This accomplishes the task of automatically cancelling CI runs on pushes to the same branch,
+  # # which is a common feature in most CI systems but currently not possible with GitHub actions.
+  # cleanup-runs:
+  #   continue-on-error: true
+  #   runs-on: ubuntu-latest
+  #   if: "!startsWith(github.ref, 'refs/tags/') && !contains(github.actor, 'renovate') && github.ref != 'refs/heads/dev' && github.ref != 'refs/heads/integration' && github.ref != 'refs/heads/patch-dev' && github.ref != 'refs/heads/latest'"
```

```suggestion
  #   if: (!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/dev' && github.ref != 'refs/heads/integration' && github.ref != 'refs/heads/patch-dev' && github.ref != 'refs/heads/latest') || contains(github.actor, 'renovate') 
```

---

Maybe GH was evaluating this as always thruthy because of the quotes - or I'm not aware that is is a "valid" conditional syntax.

---

Thanks not sure what happened here I'll figure out in another PR later
```suggestion
  # contains(github.actor, 'renovate') 
  #   if: (!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/dev' && github.ref != 'refs/heads/integration' && github.ref != 'refs/heads/patch-dev' && github.ref != 'refs/heads/latest')
```